WooCommerce/Classes/POS/Presentation/ItemCardView.swift

```diff
@@ -16,7 +16,6 @@ struct ItemCardView: View {
                 ProductImageThumbnail(productImageURL: URL(string: imageSource),
                                       productImageSize: Constants.productCardHeight,
                                       scale: scale,
-                                      productImageCornerRadius: Constants.productImageCornerRadius,
```

Comment:
We do not need corner radius on the product image, we have one for the whole view